index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/font-awesome/css/font-awesome.min.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="flex-nav">
+        <h1 class="logo">My Logo</h1>
+        <div class="toggleNav"><i class="fa fa-bars"></i></div>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h1>I&rsquo;m Francis.<br />Glad to see you!</h1>
+        <p>
+          I&rsquo;m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don&rsquo;t hesitate to contact
+          me.
+        </p>
+        <p class="connect">LET’S CONNECT</p>
+        <ul class="social">
+            <li><i class="fa fa-twitter"></i></li>
+            <li><i class="fa fa-linkedin"></i></li>
+            <li><i class="fa fa-medium"></i></li>
+            <li><i class="fa fa-github"></i></li>
+            <li><i class="fa fa-angellist"></i></li>
+        </ul>
```

Comment:
Kindly use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag). 👍💯

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 💪


Also, the icons are not displaying on the page. please, kindly fix that 🛠

Your design
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/6186782c-afd2-4608-b758-bffcf52a71e5)

Figma
![image](https://github.com/lukademinnosa/module1_project1/assets/78251927/39ba10de-82fd-4228-bda0-f54155c11496)